node/tests/ui_gateway_test.rs

```diff
@@ -54,6 +57,7 @@ fn ui_requests_something_and_gets_corresponding_response() {
 
 #[test]
 fn log_broadcasts_are_correctly_received_integration() {
+    thread::sleep(Duration::from_secs(5));
```

Comment:
Spanek hned na zacatku testu. Neni to k nicemu. 

Jestli tohle vyresilo problem s padajicim testem, jak jsi mi rikal, tak nejspis ty dva testy v tomhle souboru se navzajem ovlivnuji. Muzes napr. pouzit nejakou formu 'guard', ktera zajisti, ze nebudou bezet najednou, ale po sobe.

Reply:
Hele nerikej mi, ze se neco stane, kdyz ten radek jednoduse smazes. By me zajimalo, jak to vypada. Jestli ten test bez toho pada, tak muzeme spolu dat call a snad to nejak vymyslim nazivo. ;)